src/Sarif.Converters/FortifyFprConverter.cs

```diff
@@ -488,6 +489,18 @@ private void ParseVulnerability(IResultLogWriter output)
                 {
                     ParseLocationsFromTraces(result);
                 }
+                else if (AtStartOfNonEmpty(_strings.DefaultSeverity))
+                {
+                    rule.DefaultConfiguration.SetProperty("Deprecated"+ _strings.DefaultSeverity, _reader.ReadElementContentAsString());
+                }
+                else if (AtStartOfNonEmpty(_strings.InstanceSeverity))
+                {
+                    rule.SetProperty("Deprecated"+ _strings.InstanceSeverity, _reader.ReadElementContentAsString());
```

Comment:
>InstanceSeverity [](start = 60, length = 16)

`InstanceSeverity` goes in the result's property bag, not in the rule's property bag.

Reply:
done